src/app/visual/action-hub/action-hub-visual.component.html

```diff
@@ -0,0 +1,20 @@
+<sky-action-hub
+  [title]="'Action Hub' | skyLibResources"
```

Comment:
We don't need the `skyLibResources` pipe here since this is just for the visual test.